metaspace_converter/to_anndata.py

```diff
@@ -92,7 +92,13 @@ def metaspace_to_anndata(
         database = DEFAULT_DATABASE
 
     # Download annotations
-    annotations = dataset.results(database=database, fdr=fdr, **annotation_filter)
+    annotations = dataset.results(
+        database=database,
+        fdr=fdr,
+        include_chem_mods=True,
```

Comment:
If we set `include_chem_mods` and `include_neutral_losses` default to True, does this change the output only if a special database/adducts is selected?

Reply:
`include_chem_mods` and `include_neutral_losses` add **two extra columns** `chemMod` and `neutralLoss` in the results dataframe. Datasets without these configured during upload have chemMod `""` and neutralLoss `""`, so that for each unique formula/adduct, there is only one row. In that case, the number of rows is preserved, so this should not break be a breaking change.

If on or both are configured, independently from the database, the primary key is now formula/adduct/chemMod/neutralLoss. That means for the same formula/adduct, there can be multiple rows for different neutral losses if METASPACE found any. This is a change, but that use case was unusable before.